package.json

```diff
@@ -85,13 +85,7 @@
       "<rootDir>/src/"
     ],
     "unmockedModulePathPatterns": [
-      "<rootDir>/node_modules/fbjs/node_modules/core-js/",
-      "<rootDir>/node_modules/fbjs-scripts/",
-      "<rootDir>/node_modules/fbjs/node_modules/promise/",
-      "<rootDir>/node_modules/fbjs/lib/(?!(ErrorUtils.js$|fetch.js$|fetchWithRetries.js$))",
-      "<rootDir>/node_modules/react/",
-      "<rootDir>/node_modules/react-dom/",
-      "<rootDir>/node_modules/react-static-container/"
+      "<rootDir>/node_modules/(?!(fbjs/lib/ErrorUtils.js$|fbjs/lib/fetch.js$|fbjs/lib/fetchWithRetries.js$))"
```

Comment:
If you keep the deleted lines here and just have 2 patterns and add `4` and `5` to the travis config, do the tests pass in both environments? It'd be nice to work in both if it's just adding the new patterns instead of replacing them.

Is there something that makes it difficult to keep the npm 2 patterns in this file?